newIDE/app/src/stories/componentStories/LeaderboardSortOptionsDialog.stories.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import muiDecorator from '../ThemeDecorator';
+import paperDecorator from '../PaperDecorator';
+import LeaderboardSortOptionsDialog from '../../GameDashboard/LeaderboardAdmin/LeaderboardSortOptionsDialog';
+
+export default {
+  title: 'LeaderboardSortOptionsDialog ',
+  component: LeaderboardSortOptionsDialog,
+  decorators: [paperDecorator, muiDecorator],
+};
+
+export const Default = () => (
+  <LeaderboardSortOptionsDialog
+    open
+    onClose={() => action('onClose')()}
```

Comment:
```suggestion
    onClose={action('onClose')}
```

is not working?

Reply:
Sometimes it is necessary to do that to prevent JS to enter an infinite loop

Reply:
True, I wonder if it's not a max depth to set to Storybook to avoid it trying to log recursive stuff (weird that they don't handle this properly).

Would be good to spend 5 minutes investigating, otherwise good to go!

Reply:
Not sure why, but it makes the story freeze/crash when clicking on Close button